spring-boot-starter-camunda-connectors/src/main/java/io/camunda/connector/runtime/inbound/lifecycle/ActiveInboundConnectorResponse.java

```diff
@@ -20,14 +20,20 @@
 import java.util.Objects;
 
 public class ActiveInboundConnectorResponse {
+
   private final String bpmnProcessId;
+  private final String elementId;
   private final String type;
   private final Map<String, Object> data;
 
   public ActiveInboundConnectorResponse(
-      final String bpmnProcessId, final String type, final Map<String, Object> data) {
+      final String bpmnProcessId,
+      final String elementId,
+      final String type,
+      final Map<String, Object> data) {
```

Comment:
Instead of generic `data` maybe consider more precise name, e.g. `connectorPayloadData` - just a thought.

Reply:
Initially, I wanted to keep this name generic on purpose because we may want to provide all kinds of data in the future. But maybe @sbuettner will also want to change it in his health checks PR.